can_tools/scrapers/official/AZ/AZ_state.py

```diff
@@ -0,0 +1,280 @@
+import requests
+import urllib.parse
+from bs4 import BeautifulSoup
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import StateDashboard
+import pandas as pd
+import json
+import re
+
+
+class ArizonaStateData(StateDashboard):
+    """
+    Fetch county level covid data from Arizona's Tableau dashboard
+    """
+
+    # Initialize
+    source = "https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php"
+    has_location = True
+    location_type = "state"
+    state_fips = 4
+    cntys = [
```

Comment:
Do we use all of the counties in the `ArizonaStateData` scraper? Or is this just state data? If we do use them, we might want to move them to a parent class that both `ArizonaStateData` and `ArizonaData` inherit from so that we only have to make changes in one place.